src/List/ListItemSecondaryAction.js

```diff
@@ -1,25 +1,12 @@
 // @flow weak
 
 import React, { PropTypes } from 'react';
-import { createStyleSheet } from 'jss-theme-reactor';
-import classNames from 'classnames';
 
-export const styleSheet = createStyleSheet('ListItemSecondaryAction', () => ({
-  secondaryAction: {
-    position: 'absolute',
```

Comment:
I still think we need some additional padding. That could do the trick.
```css
padding-right: 8px;
```

Look at the diff of the image. 
![capture d ecran 2017-01-12 a 21 11 40](https://cloud.githubusercontent.com/assets/3165635/21906517/e2bc60e2-d90b-11e6-85b7-cdb73355f8df.png)

@neoziro That's a really great illustration that storing baseline images under the GitHub file system if far from optimal. We could have missed that regression looking at the screenshot too quickly.